docs/management/admin/endpoint-protection-rules.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[endpoint-protection-rules]]
+= Endpoint protection rules
+
+Endpoint protection rules are prebuilt rules designed to help you manage and respond to alerts generated by {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention. These rules include the <<endpoint-security, {elastic-defend}>> rule as well as additional detection and prevention rules for different {elastic-defend} protection features.
+
+NOTE: To receive {elastic-endpoint} alerts, you must install the Endpoint agent on your hosts (see <<install-endpoint>>).
+
+[discrete]
+== {elastic-defend} rule
+
+The {elastic-defend} rule automatically creates an alert from all incoming {elastic-endpoint} alerts. When this rule is enabled, the following Endpoint events are displayed as detection alerts:
+
+** Malware Prevention Alert
+** Malware Detection Alert
```

Comment:
Good point, I moved this content over from its [current place](https://www.elastic.co/guide/en/security/current/detection-engine-overview.html), but I'm not sure how up-to-date it is.

Another question is, since we've already been listing the names of these detection alerts generated by the Elastic Defend rule, should we now also list the names of the alerts generated by the new endpoint rules? We also noticed while testing that those alert names are currently the same regardless of whether they come from the Elastic Defend rule or the new rules. So if we decide to list them, it could be helpful to explain that, in order to understand which rule the alert was generated from, users need to open the rule details page (I think?).

It would be good to hear @caitlinbetz's or @approksiu's thoughts on this, too.

Reply:
The alert name is taken from the original Endpoint Alert name, and overwrites the prebuilt rule name.